app/views/partials/GenericRLSPartial.scala.html

```diff
@@ -35,4 +35,4 @@
 
 <p id="update-details-content" class="govuk-body">@{Html(Messages.NotEligible.`You must update your details with HMRC`(tellHMRCChangeDetailsLink).show)}</p>
 
-<p id="call-us-content" class="govuk-body">@{Html(Messages.NotEligible.`Call us on 0300 123 1813 as you may be able to set up a plan over the phone`.show)}</p>
+<p id="call-us-content" class="govuk-body">@{Html(Messages.NotEligible.`Call us on 0300 322 7835 as you may be able to set up a plan over the phone`.show)}</p>
```

Comment:
just to double check, is the phone number changing for all the regimes? If it is, is the old message still being used?

Reply:
if it's changing for one, I don't see why it wouldn't for the others; though I have now realised that the welsh needs to be 0300 200 1900 according to the spec so will change that, at least